src/NuGet.Core/NuGet.Resolver/Strings.resx

```diff
@@ -129,6 +129,9 @@
   <data name="FatalError" xml:space="preserve">
     <value>A fatal error occured while resolving dependencies.</value>
   </data>
+  <data name="InvalidPackageConfig" xml:space="preserve">
+    <value>Existing package.config is invalid and contains unresolved dependency '{0}'.</value>
```

Comment:
This message is good, but I suggest adding more text here to make it clear why this warning is coming up and when it can be ignored. Since it would only be shown in rare cases the length shouldn't be an issue.

``One or more unresolved package dependency constraints detected in the existing packages.config file. All dependency constraints must be resolved to add or update packages. If these packages are being updated this message may be ignored, if not the following error(s) may be blocking the current package operation: '{0}'.``